src/LibraryManager.Vsix/UI/Models/InstallDialogViewModel.cs

```diff
@@ -311,36 +311,44 @@ private static void SetNodeOpenStates(PackageItem item)
             item.IsExpanded = shouldBeOpen;
         }
 
-        public bool NoFilesSelected
+        public bool AnyFileSelected
         {
-            get { return _noFilesSelected; }
-            set { Set(ref _noFilesSelected, value); }
+            get { return _anyFileSelected; }
+            set { Set(ref _anyFileSelected, value); }
         }
 
         private bool CanInstallPackage()
         {
-            if (DisplayRoots != null)
+            if (!_isInstalling)
```

Comment:
>if (!_isInstalling) [](start = 12, length = 19)

Do you think it would be cleaner to do this (the answer might well be no):

if (_isInstalling)
{
    return false;
}

AnyFileSelected = IsAnyFileSelected(DisplayRoots);
return AnyFileSelected;

Reply:
Yes, that looks cleaner. Updated

---
In reply to: [194574761](https://github.com/aspnet/LibraryManager/pull/126#discussion_r194574761) [](ancestors = 194574761)